level-ws.js

```diff
@@ -33,7 +33,7 @@ WriteStream.prototype._write = function (d, enc, next) {
       self._buffer.length > self._options.maxBufferLength) {
     self.once('_flush', next)
   } else {
-    if (self._buffer.length === 0) {
+    if (!self._flushing) {
```

Comment:
This makes more sense too, that you check the boolean and if not set, you set it and do stuff.

Reply:
Yea. Can you think of any negative consequences?

Reply:
And I was wrong about 2. This is the simplest fix.

Reply:
> Can you think of any negative consequences?

`level-ws` isn't advertised as a performant writestream so I guess we're fine